goshopify.go

```diff
@@ -419,7 +418,8 @@ func (c *Client) logResponse(res *http.Response) {
 	if res == nil {
 		return
 	}
-	c.log.Debugf("RECV %d: %s", res.StatusCode, res.Status)
+	c.log.Debugf("Request Id: %s", res.Header.Get("X-Request-Id"))
```

Comment:
Went back and forth on the format and settled on this

Reply:
What other options did you consider?

Reply:
Maybe make this `Shopify X-Request-Id: %s` to make it clear it's the Shopify request ID ?

Reply:
> What other options did you consider?

- `RECV Request Id XXXXX-XXXXX-...`
- `RECV 200: 200 OK (Request Id XXXXX-XXXXX-...)`
- `RECV 200: 200 OK, Request Id XXXXX-XXXXX-...`

> Maybe make this Shopify X-Request-Id: %s to make it clear it's the Shopify request ID ?

That works too but I would ask: what other ID could it be? But overall it does make it _crystal_ clear so, fine with me!

Reply:
:-)
I hear you about what else it could be.
But in a more complex stack there are all sort s of request IDs (like proxy or LB request IDs) so might just be explicit and spell it out.

Once that's sorted this should be good to go!